databricks/koalas/tests/test_numpy_compat.py

```diff
@@ -103,3 +107,19 @@ def test_np_spark_compat(self):
                         np_func(pdf.a, 1), np_func(kdf.a, 1), almost=True)
                 except Exception as e:
                     raise AssertionError("Test in '%s' function was failed." % np_name) from e
+
+        # Test only top 5 for now. 'compute.ops_on_diff_frames' option increases too much time.
+        try:
+            set_option('compute.ops_on_diff_frames', True)
+            for np_name, spark_func in list(binary_np_spark_mappings.items())[:5]:
```

Comment:
BTW, I checked all the tests pass for its complete list.

Reply:
I checked it in my local as well.